Plugwise_Smile/Smile.py

```diff
@@ -317,12 +318,13 @@ async def update_direct_objects(self):
     async def update_domain_objects(self):
         """Request domain_objects data."""
         new_data = await self.request(DOMAIN_OBJECTS)
+        url = f"{self._endpoint}{DOMAIN_OBJECTS}"
```

Comment:
Is this added because of this line? 
```
                _LOGGER.info(
                    "Plugwise System Error Notification present but unable to process, manually investigate: %s",
                    url,
)
````

Reply:
Yep